builtins/target-avx2-i64x4.ll

```diff
@@ -1,52 +1,72 @@
-;;  Copyright (c) 2013-2023, Intel Corporation
+;;  Copyright (c) 2013-2025, Intel Corporation
 ;;
 ;;  SPDX-License-Identifier: BSD-3-Clause
 
 define(`HAVE_GATHER', `1')
 define(`ISA',`AVX2')
+define(`WIDTH',`4')
+define(`MASK',`i64')
+include(`util.m4')
 
-include(`target-avx1-i64x4base.ll')
+declare void @__masked_store_blend_i32(<4 x i32>* nocapture, <4 x i32>, 
+                                      <4 x i64>) nounwind alwaysinline
+declare void @__masked_store_blend_i64(<4 x i64>* nocapture , <4 x i64>,
+                                      <4 x i64>) nounwind alwaysinline
+declare i64 @__movmsk(<4 x i64>) nounwind readnone alwaysinline
 
-rdrand_definition()
-
-;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
-;; svml
-
-include(`svml.m4')
-svml(ISA)
-
-;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
-;; int min/max
-
-;; declare <4 x i32> @llvm.x86.sse41.pminsd(<4 x i32>, <4 x i32>) nounwind readnone
-;; declare <4 x i32> @llvm.x86.sse41.pmaxsd(<4 x i32>, <4 x i32>) nounwind readonly
-
-define <4 x i32> @__min_varying_int32(<4 x i32>, <4 x i32>) nounwind readonly alwaysinline {
```

Comment:
Min/max function are used from generic targets, right? I don't see definitions in avx1 target.

Reply:
No, they are there in `target-avx1-i64x4.ll` almost in the end of the file (line 573).

Reply:
I see now, they are separated from min/max for double and I didn't notice them.